### PR TITLE
Installed react-router-dom

### PR DESCRIPTION
**Pull request by Peacesandy**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### CODE REVIEW

**Comment by Peacesandy** [comment]
Installed react router dom

**Comment by naura1835** [comment]
Hello @Peacesandy if you can please make your PR a little bit descriptive, it will be easier to get a sense of what issue was resolved. Other than that everything looks good. Happy coding!

**Comment by a-maffei** [comment]
Hi @iblas1 and @Peacesandy, I also checked the branch code and everything looks great. 

For next time, could you please add some information to the PR following the template and also tag the other team whenever you're done with your work? Either here or on Slack. It helps keep track of our collaboration and make sure we don't miss anything. Thank you!

**Comment by Peacesandy** [comment]
> Hello @Peacesandy if you can please make your PR a little bit descriptive, it will be easier to get a sense of what issue was resolved. Other than that everything looks good. Happy coding!

Okay, Thank you for your observation

File: src/views/Layout.jsx [hunk 1]
```diff
@@ -21,15 +21,15 @@ export function Layout() {
...
+					</NavLink>
+					<NavLink to={'/list'} className="Nav-link">
 						List
-					</a>
-					<a href="#" className="Nav-link">
+					</NavLink>
+					<NavLink to={'/add-item'} className="Nav-link">
 						Add Item
-					</a>
+					</NavLink>
```
Comment (Amaka202) [inline src/views/Layout.jsx:32]: Nice job team. your code was straightforward!

**Review by Amaka202 — APPROVED** [review]
Your ACs were met but I have a few observations as regards this task.

- I noticed the PR was made even before the functionality was ready. I noticed @Peacesandy added react router dom and made a PR before @iblas1 started adding the Navlink. The correct order is work on your feature, push to github, your team partner pulls down your code, works on their own part and push back, then when you both have certified that your task is done, you make a PR.
- You didn't use our branch naming convention to name your branch. which is: the two pairs' initials and the task description separated by a dash eg ps-ia-make-nav-work.
- You didn't tag the other team using github's 'reviewer' functionality.
- The PR wasn't descriptive enough as the other team noticed too. We have an easy template to follow once you are trying to raise a PR.

It is a lot I know😫. Don't worry with practice you will get used to it. These practices just help your teammates and your future self understand what the branch/code is about. Thankfully we will have a code review learning module tomorrow and we will explain better. These are obviously non-blocking observations so you can go ahead to merge your code. 💥💥

Well done team!✅

**Pull request merged**